docs/source/tutorials/building_blocks.rst

```diff
@@ -0,0 +1,342 @@
+	   self.play(FadeIn(square))
+
+	   # ... some move or rotate mobjects around...
+	   self.play(Rotate(square, PI/4))
+
+	   # some animations remove mobjects from the screen
+	   self.play(FadeOut(square))
+
+           self.wait(1)
+
+.. image:: ../_static/building_blocks/some_animations.gif
+    :align: center
+    :alt: a square fading it, rotating, and fading out
+
+Put simply, animations are procedures that interpolate between two mobjects.
+For example, :code:`FadeIn(square)` starts with a fully transparent version of
+:code:`square` and ends with a fully opaque version, interpolating between them
+by gradually increasing the opacity.  :class:`.FadeOut` works in the opposite
+way: it interpolates from fully opaque to fully transparent.  As another
+example, :class:`.Rotate` starts with the mobject passed to it as argument, and
+ends with the same object but rotated by a certain amount, this time
+interpolating the mobject's angle instead of its opacity.
+
+
+Animating methods
+=================
+
+Any property of a mobject that can be changed can be animated.  In fact, any
+method that changes a mobject's property can be used as an animation, through
+the use of :class:`.ApplyMethod`.
+
+.. code-block:: python
+
+   class ApplyMethodExample(Scene):
+       def construct(self):
+           square = Square().set_fill(RED, opacity=1.0)
+           self.add(square)
+
+	   # animate the change of color
+	   self.play(ApplyMethod(square.set_fill, WHITE))
+	   self.wait(1)
+
+	   # animate the change of position
+	   self.play(ApplyMethod(square.shift, UP))
+           self.wait(1)
+
+.. image:: ../_static/building_blocks/apply_method.gif
+    :align: center
+    :alt: animation of a square changing color, then moving up
+
+:meth:`.ApplyMethod` receives one mandatory argument which is the method of the
+mobject to animate (e.g. :code:`square.set_fill` or :code:`square.shift`), and
+any number of optional arguments which are then passed to the method call.  For
+example, :code:`ApplyMethod(square.shift, UP)` executes
+:code:`square.shift(UP)`, but animates it instead of applying it immediately.
+
+Animation run time
+==================
+
+By default, any animation passed to :meth:`play` lasts for exactly one second.
+Use the :code:`run_time` argument to control the duration.
+
+.. code-block:: python
+
+   class RunTime(Scene):
+       def construct(self):
+           square = Square()
+           self.add(square)
+	   self.play(ApplyMethod(square.shift, UP), run_time=3)
+	   self.wait(1)
+
+.. image:: ../_static/building_blocks/run_time.gif
+    :align: center
+    :alt: a square moving up very slowly
+
+
+******
+Scenes
+******
+
+The :class:`.Scene` class is the connective tissue of manim.  Every mobject has
+to be :meth:`~.Scene.add` -ed to a scene to be displayed, or
+:meth:`~.Scene.remove` -d from it to cease being displayed.  Every animation
+has to be :meth:`~.Scene.play` -ed by a scene, and every time interval where no
```

Comment:
Superfluous spaces.

```suggestion
to be :meth:`~.Scene.add`-ed to a scene to be displayed, or
:meth:`~.Scene.remove`-d from it to cease being displayed.  Every animation
has to be :meth:`~.Scene.play`-ed by a scene, and every time interval where no
```

Reply:
I tried this, but I don't think it renders correctly. I think rst requires a space after each backtick `. Any workarounds?

Reply:
Are you sure? In renders just fine if I try it locally.

Otherwise, its always possible to write custom text for the links, for example:

```
:meth:`added <.Scene.add>`
```

Reply:
D'oh!